atd-etl/app/process/socrata_queries.py

```diff
@@ -94,15 +86,7 @@
                 ],
                 _and: {
                     crash: {crash_date: {_lt: "$date_limit", _gte: "$initial_date_limit" }}
-                    _or: [
-                        {crash: {austin_full_purpose: {_eq: "Y"}}},
-                        {
-                            _and: [
-                                {crash: {city_id: {_eq: 22}}},
-                                {crash: {position: {_is_null: true}}} 
-                            ]
-                        }
-                    ]
+                    crash: {in_austin_full_purpose: { _eq: true}}
```

Comment:
these queries look rly confusing to me, i cant tell if this is right

Reply:
These trip me up too! I'm reading this update as crash date between the two date vars AND crash in AFP. Previously, it was crash date between the two date vars AND [in AFP (jurisdiction = 5) OR (city_id = 22 AND position not null)]. 

The new definition of `in_austin_full_purpose` covers everything in the brackets so that looks good to me. 👍